agent/05_agent_create_cluster.sh

```diff
@@ -61,10 +66,22 @@ function force_mirror_disconnect() {
 
 }
 
+function enable_assisted_service_ui() {
+  node0_ip=$(get_node0_ip)
+  ssh_opts=(-o 'StrictHostKeyChecking=no' -o 'UserKnownHostsFile=/dev/null' -q core@${node0_ip})
+
+  until ssh "${ssh_opts[@]}" "[[ -f /run/assisted-service-pod.pod-id ]]"
+  do
+    echo "Waiting for node0"
+    sleep 5s;
+  done
+
+  ssh "${ssh_opts[@]}" "sudo /usr/bin/podman run -d --name=assisted-ui --pod-id-file=/run/assisted-service-pod.pod-id quay.io/edge-infrastructure/assisted-installer-ui:latest"
```

Comment:
It does not. It could perfectly be run in the hypervisor. I'll create a task to do that with the relevant information. For now though, let's merge this

Reply:
https://issues.redhat.com/browse/AGENT-311